.github/workflows/changelog-and-release.yml

```diff
@@ -28,7 +28,7 @@ on:
       add_date:
         description: 'Add date to version number in changelog and news. ie. "v1.0.1 (2021-7-17)" [true|false]'
         required: true
-        default: 'true'
+        default: 'false'
```

Comment:
also this must be kept as "true"
this has effect not only on the old xml "news" section, but also on changelog.txt

Reply:
Okay, will change.

Reply:
done